common/azure-common/src/main/java/com/azure/common/exception/AzureException.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.azure.common.exception;
+
+/**
+ * The base exception type for all Azure-related exceptions.
+ *
+ * @see ServiceRequestException
+ */
+public class AzureException extends RuntimeException {
+
+    /**
+     * Initializes a new instance of the AzureException class.
+     *
+     * @param message the exception message or the response content if a message is not available
```

Comment:
Should we delete "or the response content if a message is not available" from description? -> `AzureException` is root exception type, so not always based on service `response`.

Reply:
Good point - I'll fix that. Thanks.